docs/reference/env-variables.rst

```diff
@@ -0,0 +1,971 @@
+.. meta::
+    :description: Environment variables reference
+    :keywords: AMD, ROCm, environment variables, environment, reference
+
+.. role:: cpp(code)
+   :language: cpp
+
+.. _env-variables-reference:
+
+*************************************************************
+Environment variables reference
+*************************************************************
+
+ROCm common and important Environment Variables
+===============================================
+
+The following table contains the environment variables that are most commonly used in the ROCm software stack. These variables can be useful for simple tasks, like building a ROCm library or running applications on AMD cards.
+
+.. list-table::
+    :header-rows: 1
+    :widths: 70,30
+
+    * - **Environment variable**
+      - **Usage**
+
+    * - | ``HIP_DIR``
+        | The path of the HIP SDK on Microsoft Windows.
+      - Default: ``C:/hip``
+
+    * - | ``HIP_PATH``
+        | The path of the HIP SDK on Microsoft Windows. On Linux the ``ROCM_PATH``
+        | environment variable used to set different ROCm root path.
+      - Default: ``C:/hip``
+
+    * - | ``ROCM_PATH``
+        | The path of the installed ROCm software stack on Linux. On Microsoft 
+        | Windows the ``HIP_DIR`` environment variable used to set 
+        | different HIP SDK path.
+      - Default: ``/opt/rocm``
+
+    * - | ``HIP_PLATFORM``
+        | The platform HIP is targeting. If ``HIP_PLATFORM`` is not set, then HIPCC
+        | will attempt to auto-detect based on if NVCC is found.
+      - ``amd``, ``nvidia``
+
+CLR Environment Variables
+=========================
+
+AMD Common Language Runtime (clr) contains source codes for AMD's compute languages runtimes: **HIP** and **OpenCL™**. The environment variables, which are affecting this library can effect **HIP** and **OpenCL™** libraries or applications too.
+
+The following table contains the environment variables affecting all backends of project clr.
+
+.. list-table::
+    :header-rows: 1
+    :widths: 70,30
+
+    * - **Environment variable**
+      - **Usage**
+
+    * - | ``HIP_PLATFORM``
+        | The platform HIP is targeting. If ``HIP_PLATFORM`` is not set, then HIPCC will attempt to auto-detect based on if NVCC is found.
```

Comment:
```suggestion
        | The platform targeted by HIP. If ``HIP_PLATFORM`` is not set, then HIPCC attempts to auto-detect the platform if it can find NVCC.
```